WordPress/WordPressTest/PostTests.swift

```diff
@@ -376,6 +376,54 @@ class PostTests: XCTestCase {
         XCTAssertFalse(revision.hasLocalChanges())
     }
 
+    func testThatVersionConflictWorksShouldNotConflict() {
+        let original = newTestPost()
+        XCTAssertFalse(original.hasVersionConflict(), "Post should return false for `hasVersionConflict`")
+    }
+
+    // Local changes alone should not trigger `hasVersionConflict`
+    func testThatVersionConflictWorksShouldNotConflictWithOnlyTagChange() {
+        let original = newTestPost()
+        let local = original
+
+        local.tags = "test"
+
+        XCTAssertFalse(local.hasVersionConflict(), "Post should return false for `hasVersionConflict`")
+    }
+
+    // dateModified changes alone should not trigger `hasVersionConflict`
+    func testThatVersionConflictWorksShouldNotConflictWithOnlyDateChanges() {
+        let original = newTestPost()
+        let local = original
+
+        local.setPrimitiveValue((Date() - 5), forKey: "dateModified")
+
+        XCTAssertFalse(local.hasVersionConflict(), "Post should return false for `hasVersionConflict`")
+    }
+
+
+    func testThatVersionConflictWorksWithPreviousPost() {
+        let original = newTestPost()
+        original.setPrimitiveValue(Date(), forKey: "dateModified")
+        let local = original.createRevision() as! Post
+
+        local.setPrimitiveValue((Date() - 5), forKey: "dateModified")
+        local.tags = "test"
+
+        XCTAssertTrue(local.hasVersionConflict(), "Local post with a previous date should return true for `hasVersionConflict`")
```

Comment:
Jen, just a minor thing. Your `act` here is the call to `local.hasVersionConflict()` itself. :)

So this could be rearranged like:

```suggestion
        // Arrange
        let original = newTestPost()
        original.setPrimitiveValue(Date(), forKey: "dateModified")
        let local = original.createRevision() as! Post
        local.setPrimitiveValue((Date() - 5), forKey: "dateModified")
        local.tags = "test"
        
        // Act
        let hasVersionConflict = local.hasVersionConflict()

        XCTAssertTrue(hasVersionConflict, "Local post with a previous date should return true for `hasVersionConflict`")
```

But don't worry about that. Just keep in mind for the next tests, it's not a big deal, really.